postgres.go

```diff
@@ -184,8 +196,12 @@ func columnMapper(column *rel.Column) (string, int, int) {
 	return typ, m, n
 }
 
-func check(err error) {
-	if err != nil {
-		panic(err)
+func init() {
+	// Identify if pgx driver is available and default to that instead.
+	for _, drv := range db.Drivers() {
+		if drv == "pgx" {
+			driverName = "pgx"
```

Comment:
nit: (maybe not in this PR) I think we should provide a way to override this behavior? such as explicitly setting the driver name?

Reply:
Thought about that but as there are currently no more options I did not do this